api/proto/disperser/disperser.proto

```diff
@@ -32,6 +32,18 @@ service Disperser {
 	// The blob should have been initially dispersed via this Disperser service
 	// for this API to work.
 	rpc RetrieveBlob(RetrieveBlobRequest) returns (RetrieveBlobReply) {}
+
+	////////////////////////////////////////////////////////////////////////////////////////////////
+	// These RPCs are a work in progress and are not yet fully functional. API may not be stable. //
+	////////////////////////////////////////////////////////////////////////////////////////////////
+
+	// Retrieves the requested blob from the disperser's backend.
+	// This RPC is is a work in progress with undefined behavior, use with caution.
+	rpc GetBlob(common.BlobKey) returns (common.BlobData) {}
+
+	// Retrieves the requested chunk from the Disperser's backend.
+	// This RPC is is a work in progress with undefined behavior, use with caution.
+	rpc GetChunk(common.ChunkKey) returns (common.ChunkData) {}
```

Comment:
Will the addition of new RPCs cause problems if I don't actually register methods to handle them? Or do I need to register simple no-op methods for safety?

Reply:
I don't know the answer to this off-hand.

Reply:
But my guess is that we would need to implement no-op methods in order for the server to even compile?

Reply:
It compiles without them. In the interest of safety, probably I will write unit tests to verify behavior with these new RPCs.